Source/Core/DolphinQt/Config/Mapping/HotkeyControllerProfile.cpp

```diff
@@ -16,10 +16,27 @@ HotkeyControllerProfile::HotkeyControllerProfile(MappingWindow* window) : Mappin
 
 void HotkeyControllerProfile::CreateMainLayout()
 {
-  m_main_layout = new QHBoxLayout();
-
-  m_main_layout->addWidget(CreateGroupBox(
-      tr("Controller Profile"), HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE)));
+  m_main_layout = new QGridLayout;
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %1").arg(1), 
+                     HotkeyManagerEmu::GetHotkeyGroup(HKGP_CONTROLLER_PROFILE_1)),
+      0, 0);
+
+  m_main_layout->addWidget(
+      CreateGroupBox(tr("Wii Remote %2").arg(2), 
```

Comment:
Perhaps this was a find/replace mistake. `%2` is probably meant to be `%1`.
Same problem below with 3 and 4.

Reply:
Then, I guess I have to switch 1 & 2, and 3 & 4 ?

Reply:
Hmm? `%1` is the format string placeholder for the first value supplied to the `.arg` function.

Reply:
Oh sure.